CO2_Gadget_EINK.h

```diff
@@ -440,10 +451,15 @@ void initDisplay(bool fastMode = false) {
     delay(2000);  // Enjoy the splash screen 2 seconds
 }
 
-void showBatteryIcon(int32_t posX, int32_t posY, bool forceRedraw) {
+bool showBatteryIcon(int32_t posX, int32_t posY, bool forceRedraw) {
+    RTC_DATA_ATTR static int16_t oldBatteryValue = -1;
+    int16_t newBatteryValue;
```

Comment:
Should it be on RTC memory?
I don't understand. What's the point for this?
Anyway it conflicts with the last changes.

Reply:
I used RTC_DATA_ATTR only becouse you used it like this in low-power-eink branch.
https://github.com/melkati/CO2-Gadget/blob/c5b093736f78e757edac4b27eee518ebd74af364/CO2_Gadget_EINK.h#L588
For me is not necessary at all.